Sanket/Task1_2_3.md

```diff
@@ -1,29 +1,30 @@
-TASK 1
+#TASK 1
 
-1.What are git and GitHub?
+#1.What are git and GitHub?
```

Comment:
Space after #. Same comments for all of the case.

Reply:
Changes have been made as per instruction given by you.